.github/workflows/run-tests.yml

```diff
@@ -81,6 +82,7 @@ jobs:
       env:
         DEPENDENCIES: ${{ matrix.dependencies }}
     - name: Test with pytest
+    	timeout-minutes: 30
```

Comment:
Can you fix the indentation here?

Reply:
Yeah, thanks for the noticing, should be ok now